cmd/console_client/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"bufio"
+	"haracho-go/internal/client"
+	"haracho-go/internal/client/console"
+	"haracho-go/internal/command"
+	"haracho-go/internal/service"
+	"os"
+)
+
+func main() {
+	command.RegisterCommands()
```

Comment:
```suggestion
	collection := new(client.CommandCollection)
	command.RegisterCommands(collection)
```
別記の通りに `RegisterCommands` を直すとこうなる